public-api/vx/node.php

```diff
@@ -28,6 +28,17 @@
 		'grade-06-out' => ['length' => 1],
 		'grade-07-out' => ['length' => 1],
 		'grade-08-out' => ['length' => 1],
+		
+		'link-01-name' => ['length' => 64],
+		'link-02-name' => ['length' => 64],
+		'link-03-name' => ['length' => 64],
+		'link-04-name' => ['length' => 64],
+		'link-05-name' => ['length' => 64],
+		'link-01-url' => ['length' => 128],
```

Comment:
https://stackoverflow.com/questions/219569/best-database-field-type-for-a-url/219664#219664